### PR TITLE
Fix scons 64bit OS detection

### PR DESCRIPTION
**Pull request by miguelportilla**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -123,7 +123,7 @@ import time
 import glob
 import SCons.Action
 
-if (platform.architecture()[0] != '64bit'):
+if (not platform.machine().endswith('64')):
     print('Warning: Detected {} architecture. Rippled requires a 64-bit OS.'.format(
           platform.architecture()[0]));
```
Comment (bachase) [inline SConstruct]: Have the error print the `platform.machine()` too?

**Comment by bachase** [comment]
:+1: aside from the error message.

**Comment by seelabs** [comment]
:+1:

**Comment by HowardHinnant** [comment]
👍 Works for me (macOS).

**Comment by nbougalis** [comment]
Merged as effd8c9737b9c5b938af885b414e994fb6025cac

**Pull request closed without merging**